android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/app/ApplicationConfiguration.kt

```diff
@@ -36,5 +36,5 @@ data class ApplicationConfiguration(
   val deviceToDeviceSync: DeviceToDeviceSyncConfig? = null,
   val snackBarTheme: SnackBarThemeConfig = SnackBarThemeConfig(),
   val reportRepeatTime: String = "",
-  val taskExpireJobRepeatIntervalConfig: Long = 60
+  val taskExpireJobRepeatInterval: Long = 60
```

Comment:
is this in minutes? I think we should add that to the name so it's clear
```suggestion
  val taskExpireJobRepeatIntervalMinutes: Long = 60
```